app/src/components/Shared/Accordion.jsx

```diff
@@ -4,21 +4,24 @@ import Rhombus from './Rhombus';
 import Loader from './Loader';
 import AccordionStyles from '../../../styles/components/shared/c-content-accordion.scss';
 import { scrollTo } from '../../lib/Utils';
-import $ from 'jquery';
 
 class Accordion extends Component {
   componentDidUpdate() {
     if (!this.props.autoscroll || this.scrolled) return;
-
-    const $el = $(`.${AccordionStyles['-opened']}`).parent();
-    if ($el.length) {
-      scrollTo($el);
-    }
     this.scrolled = true;
   }
 
-  onItemClick(index, slug) {
-    this.props.onAccordionItemClick(index, slug, this.props.accordionId);
+  onItemClick(index, slug, id) {
+    if (window.innerWidth <= 768) {
+      this.props.onAccordionItemClick(index, slug, this.props.accordionId);
+      setTimeout(() => {
+        const el = document.getElementById(id);
```

Comment:
This is useless. You can pass the onClick callback the element via `e.currentTarget` so you don't have to search for it. Don't forget to remove the "title" param you added.